docs/network-flow-visibility.md

```diff
@@ -710,34 +745,53 @@ serviceTemplates:
           port: 9000
 ```
 
-This service is also used by the Flow Aggregator and Grafana. If you update the
-HTTP port, please update `url` in [datasource_provider.yml][grafana_datasouce_provider_yaml].
-If you update the TCP port, please update `jsonData.port` in [datasource_provider.yml][grafana_datasouce_provider_yaml]
-and `databaseURL` in the [Flow Aggregator Configuration](#configuration-1).
+This service is used by the Flow Aggregator and Grafana.
+
+- If you have changed the HTTP port, please update the `url` of a resource of name `grafana-datasource-provider` in `flow-visibility.yml`.
+
+- If you have changed the TCP port, please update the `databaseURL` following [Flow Aggregator Configuration](#configuration-1),
+and also update the `jsonData.port` of the `grafana-datasource-provider` resource.
+
+```yaml
+apiVersion: v1
+data:
+  datasource_provider.yml: |
+    apiVersion: 1
+    datasources:
+      - name: ClickHouse
+        type: grafana-clickhouse-datasource
+        access: proxy
+        url: http://clickhouse-clickhouse.flow-visibility.svc:8123
+        editable: true
+        jsonData:
+          server: clickhouse-clickhouse.flow-visibility.svc
+          port: 9000
+          username: $CLICKHOUSE_USERNAME
+        secureJsonData:
+          password: $CLICKHOUSE_PASSWORD
+kind: ConfigMap
+metadata:
+  name: grafana-datasource-provider-h868k56k95
+  namespace: flow-visibility
+```
 
 The ClickHouse throughput depends on two factors - the storage size of the ClickHouse
 and the time interval between the batch commits to the ClickHouse. Larger storage
 size and longer commit interval provide higher throughput.
 
 Grafana flow collector supports the ClickHouse in-memory deployment with limited
-storage size. This is specified in [clickhouse.yml][clickhouse_manifest_yaml].
-The default value of storage size for the ClickHouse server is 8 GiB. Users
+storage size. This is specified in `flow-visibility.yml` under the `ClickHouseInstallation`
```

Comment:
line wrap

Reply:
This a bit pedant, but maybe you can consider writing:

[...] under the `clickhouse` resource of kind `ClickHouseInstallation`. The default value [...]